src/test/java/tutorease/address/logic/parser/FindLessonCommandParserTest.java

```diff
@@ -0,0 +1,35 @@
+package tutorease.address.logic.parser;
+
+import static tutorease.address.logic.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static tutorease.address.logic.parser.CommandParserTestUtil.assertParseFailure;
+import static tutorease.address.logic.parser.CommandParserTestUtil.assertParseSuccess;
+
+import java.util.Arrays;
+
+import org.junit.jupiter.api.Test;
+
+import tutorease.address.logic.commands.FindLessonCommand;
+import tutorease.address.model.lesson.LessonContainsNamesPredicate;
+
+public class FindLessonCommandParserTest {
+
+    private FindLessonCommandParser parser = new FindLessonCommandParser();
+
+    @Test
+    public void parse_emptyArg_throwsParseException() {
+        assertParseFailure(parser, "     ", String.format(MESSAGE_INVALID_COMMAND_FORMAT,
+                FindLessonCommand.MESSAGE_USAGE));
+    }
```

Comment:
Maybe besides empty argument, more types of imvalid arguments could be tested? eg. null, find integers in a name

Reply:
added for null and find integers